docker_services_cli/config.py

```diff
@@ -105,8 +105,14 @@
         "REDIS_7_LATEST": "7",
     },
     "CONTAINER_CONNECTION_ENVIRONMENT_VARIABLES": {
-        "mq": {"BROKER_URL": "redis://localhost:6380/0"},
-        "cache": {"CACHE_TYPE": "redis"},
+        "mq": {
+            "BROKER_URL": "redis://localhost:6380/0",
+            "CACHE_REDIS_URL": "redis://127.0.0.1:6380/0",
```

Comment:
```suggestion
```

minor: do we need that for `mq`/Celery? I remember there's something about a results backend for Celery, but that's maybe under a different env/config?